Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/ResourceClasses/Constants.java

```diff
@@ -0,0 +1,9 @@
+package org.firstinspires.ftc.team6220_PowerPlay.ResourceClasses;
+
+public class Constants {
+    public double WheelDiam = 3.73;
+    public double WheelCirc = 11.71;
+    public double TicksPerRev = 537.6;
```

Comment:
You should consider replacing this magic number 537.6 with an expression for ticks per revolution, based on the motor's gear ratio.  That way, the magic number changes if you change to motors with a different gear ratio.